mixins/_typography.pcss

```diff
@@ -286,14 +286,34 @@
  * ----------------------------------------------------------------------------- */
 
 @define-mixin anchor-thin-alt {
-	border-bottom: 1px solid var(--color-accent-primary);
-	color: var(--color-text-primary);
-	transition: var(--transition-color);
+	border-bottom: 1px solid var(--tec-color-link-accent);
+	color: var(--tec-color-link-primary);
+	transition: var(--tec-transition-color);
 
 	&:active,
 	&:focus,
 	&:hover {
-		border-bottom: 1px solid var(--color-accent-primary);
-		color: var(--color-accent-primary);
+		border-bottom: 1px solid currentColor;
+		color: var(--tec-color-link-accent);
+	}
+}
+
+/* -----------------------------------------------------------------------------
+ * Anchor - Event Title
+ * ----------------------------------------------------------------------------- */
+@define-mixin event-title-link {
+	color: var(--tec-color-text-events-title);
+
+	&:active,
+	&:focus,
+	&:hover,
+	&:visited {
+		color: var(--tec-color-text-events-title);
+	}
+
+	&:active,
+	&:focus,
+	&:hover {
+		border-color: currentColor;
 	}
```

Comment:
I would actually do this the other way. If the class name this is attached to is long, and it'll likely be longer than the `border-color: currentColor` style, duplicating a selector will be more bloat than duplicating a style.

```
&:active,
&:focus,
&:hover {
	border-color: currentColor;
	color: var(--tec-color-text-events-title);
}

&:visited {
	color: var(--tec-color-text-events-title);
}
```